Dockerfile

```diff
@@ -0,0 +1,75 @@
+# Multi-stage build for Claude Code UI
+
+# Stage 1: Build frontend
+FROM node:20-alpine AS frontend-builder
+
+WORKDIR /app
+
+# Copy package files
+COPY package*.json ./
+
+# Install dependencies
+RUN npm ci --only=production
```

Comment:
@krzemienski is `--only=production` needed here? We need devDependencies (vite, tailwind, postcode) before we can do stage 2 build, no?